subprojects/liblzma.wrap

```diff
@@ -0,0 +1,13 @@
+[wrap-file]
+directory = xz-5.2.1
+
+source_url = http://tukaani.org/xz/xz-5.2.1.tar.xz
+source_filename = xz-5.2.1.tar.xz
+source_hash = 6ecdd4d80b12001497df0741d6037f918d270fa0f9a1ab4e2664bf4157ae323c
+
+patch_url = http://public.kymeria.fr/KIWIX/xz-5.2.1-wrap.zip
+patch_filename = xz-5.2.1-wrap.zip
```

Comment:
Isn't it better/safer to host these files on openzim.org (same for `subprojects/zstd.wrap`)?

Reply:
Definitly, if not hosted/maintainted by Meson itself, I would like to see this in our "kiwix-build" git, and without compression/packaging/zip.

Reply:
Yes, I've put the file on `https://download.kiwix.org/dev`
Last push just change the `patch_url`